portal-api/src/constants/file-upload.js

```diff
@@ -1,9 +1,17 @@
 const FILE_UPLOAD = {
   ALLOWED_FORMATS: ['.bmp', '.doc', '.docx', '.gif', '.jpeg', '.jpg', '.pdf', '.png', '.ppt', '.pptx', '.tif', '.txt', '.xls', '.xlsx'],
+  ALLOWED_FORMATS_UTILISATION_REPORT: ['.csv'],
```

Comment:
This is because portal we can upload .xlsx or .csv files but we convert .xlsx to .csv there and so only worry about .csv files from portal-api onwards (and so we only store csv files in Azure file storage for security reasons)